test/bemhtml-test.js

```diff
@@ -104,4 +104,15 @@ describe('BEMHTML engine tests', function() {
       block: 'b1'
     }, '<div class="b1">ok</div>');
   });
+
+  describe('omit optional', function() {
+    it('closing tags', function() {
```

Comment:
describe in optional mode
it should omit optional tags ← current test
it should not omit not optional (opposing) ← missed